lib/person.rb

```diff
@@ -1 +1,80 @@
-# your code goes here
+class Person
+  attr_reader :name
+  attr_accessor :bank_account, :happiness, :hygiene
+
+  def initialize(
+    name,
+    bank_account = 25,
+    happiness = 8,
+    hygiene = 8
+    )
+    @name = name
+    @bank_account = bank_account
+    @happiness = happiness
+    @hygiene = hygiene
+  end
```

Comment:
```suggestion
```suggestion
  def initialize (name)
    @name = name
    @bank_account = 25
    @happiness = 8
    @hygiene = 8
  end
```
```